opennms-config-model/src/main/java/org/opennms/netmgt/xml/eventconf/Events.java

```diff
@@ -70,10 +37,19 @@
 import org.springframework.orm.ObjectRetrievalFailureException;
 import org.springframework.util.StringUtils;
 
-@XmlRootElement(name="events")
+import javax.xml.bind.annotation.*;
```

Comment:
Don't use wildcard imports. You can change this setting in IntelliJ. 
It's fine for now but I generally avoid formatting whole file but only format added sections that way reviewer can concentrate on what changed.

Reply:
Got it, thank you. Will toggle that setting and avoid whole-file formatting in the future.

Reply:
removing wildcard imports is not optional since OpenNMS code doesn't use wildcard imports.
 formatting is fine for now.

Reply:
Removed them by changing IntelliJ's settings and running "optimize imports".